src/platform/user/profile/vet360/components/base/VAPEditModalActionButtons.jsx

```diff
@@ -0,0 +1,142 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import AlertBox from '@department-of-veterans-affairs/formation-react/AlertBox';
+import { toLower } from 'lodash';
+
+import recordEvent from 'platform/monitoring/record-event';
+import LoadingButton from 'platform/site-wide/loading-button/LoadingButton';
+
+class VAPEditModalActionButtons extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      deleteInitiated: false,
+    };
+  }
+
+  componentDidUpdate(prevProps, prevState) {
+    // Once the AlertBox is mounted, we want to set the focus to the heading
+    // for screen reader use
+    if (this.state.deleteInitiated && !prevState.deleteInitiated) {
+      const heading = document.getElementById('deleteConfirmationHeading');
+      if (heading) {
+        heading.focus();
+      }
+    }
+    // If delete is cancelled, put focus back on modal close button
+    if (!this.state.deleteInitiated && prevState.deleteInitiated) {
+      const closeButton = document.getElementsByClassName('va-modal-close')[0];
+      if (closeButton) {
+        closeButton.focus();
+      }
+    }
+  }
+
+  cancelDeleteAction = () => {
+    this.setState({ deleteInitiated: false });
+    recordEvent({
+      event: 'profile-navigation',
+      'profile-action': 'cancel-delete-button',
+      'profile-section': this.props.analyticsSectionName,
+    });
+  };
+
+  confirmDeleteAction = e => {
+    e.preventDefault();
+    recordEvent({
+      event: 'profile-navigation',
+      'profile-action': 'confirm-delete-button',
+      'profile-section': this.props.analyticsSectionName,
+    });
+    this.props.onDelete();
+  };
+
+  handleDeleteInitiated = () => {
+    recordEvent({
+      event: 'profile-navigation',
+      'profile-action': 'delete-button',
+      'profile-section': this.props.analyticsSectionName,
+    });
+    this.setState({ deleteInitiated: true });
+  };
+
+  renderDeleteAction() {
+    if (this.props.deleteEnabled) {
+      return (
+        <a
```

Comment:
I do think that this should be a `button` that's styled to look like a text link, since its action does not direct you to a new page. I really had no idea how to decide if something should be a `button` or `a` until very recently, but I think `button` is a better semantic choice here.

Reply:
Learned something new, thanks! Will update 👍